src/schemas.jl

```diff
@@ -110,13 +114,33 @@ function Base.showerror(io::IO, e::UnknownSchemaVersionError)
               This generally indicates that this schema has not been declared (i.e.
               the corresponding `@schema` and/or `@version` statements have not been
               executed) in the current Julia session.
+              """)
+    println(io)
+
+    if !ismissing(e.schema_provider_name)
+        provider_string = string(e.schema_provider_name)
+        if !ismissing(e.schema_provider_version)
+            provider_string *= string(" (version: ", e.schema_provider_version, ")")
+        end
+        print(io, """
+                The table's metadata indicates that the table was created with a schema defined in:
 
-              In practice, this can arise if you try to read a Legolas table with a
-              prescribed schema, but haven't actually loaded the schema definition
-              (or commonly, haven't loaded the dependency that contains the schema
-              definition - check the versions of loaded packages/modules to confirm
-              your environment is as expected).
+                $(provider_string)
```

Comment:
Indented this by 2-spaces:
```suggestion
                  $(provider_string)
```